client/src/views/node.tsx

```diff
@@ -84,7 +84,7 @@ export default class NodeView extends Base<Props, State> {
                 </ChartsContainer>
 
                 <div className='contentPanel'>
-                    {!item ? <Loading /> : (
+                    {!item ? <Loading /> :  item.status ? (
```

Comment:
Do you intend to use `&&` rather than `?` here?

Reply:
Yeah I hate that style with JSX, but it was intentional.  The [docs](https://reactjs.org/docs/conditional-rendering.html) state boolean logic for on-off things and ternaries for if/else things.  This is a nested ternary so that:

1. If item is not truthy render `<Loading />`
2. Else if item.status is truthy, render the JSX
3. Else render null

Reply:
That makes sense. Will merge

Reply:
Lint actually complained about nested ternary so I updated I split it into to separate calls instead.